### PR TITLE
Add Address generator & parser

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: src/Tippy.Core/Address/Address.cs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using Tippy.Util;
+using Types = Ckb.Rpc.Types;
+
+namespace Tippy.Core.Address
+{
+    public class Address
+    {
+        static readonly string SECP_CODE_HASH = "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8";
```
Comment (ashchan) [inline src/Tippy.Core/Address/Address.cs:11]: Please follow dot net naming convention. Constant (field) should use `CamelCase`.

File: src/Tippy.Core/Address/Bech32.cs [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            }
+            if (!lower)
+            {
+                return (hrp.ToUpper(), data.Take(data.Count - 6).ToArray());
+            }
+            return (hrp, data.Take(data.Count - 6).ToArray());
+        }
+    }
+
+    public class ConvertAddresse
```
Comment (ashchan) [inline src/Tippy.Core/Address/Bech32.cs:157]: `ConvertAddress`?

File: src/Tippy.Util/Hex.cs [hunk 3]
```diff
@@ -22,5 +23,19 @@ public static BigInteger HexToBigInteger(string hex)
         }
 
         public static string UInt64ToHex(UInt64 num) => $"0x{num:x}";
+
+        public static byte[] HexToBytes(string i)
```
Comment (ashchan) [inline src/Tippy.Util/Hex.cs:27]: Does this convert hex format number, or just hex string (like private key) to bytes? If it's for string, not for number, we can use `System.Convert`'s `FromHexString` and `ToHexString`.

File: src/Tippy.Core/Address/Address.cs (hunk 1 above)
Comment (classicalliu) [reply src/Tippy.Core/Address/Address.cs]: updated

File: src/Tippy.Core/Address/Bech32.cs (hunk 2 above)
Comment (classicalliu) [reply src/Tippy.Core/Address/Bech32.cs]: Yes, updated now.

File: src/Tippy.Util/Hex.cs (hunk 3 above)
Comment (classicalliu) [reply src/Tippy.Util/Hex.cs]: Yes, it's for string, and updated now.

**Review by ashchan — APPROVED** [review]

**Pull request merged**